packages/core/src/fetch.ts

```diff
@@ -86,7 +98,15 @@ export const fetchAds = async ({
           method: 'POST',
           errorId: `${fetchAdsError.name}`,
         })
-        reject(fetchAdsError)
+        // Try to render hardcoded fallback Ads
+        try {
+          const fallbackResponse = mapResponseToPlacementsWithContent(getFallbackAds(pendingPlacements), pendingPlacements)
```

Comment:
Same comment here. I think we can eliminate this `try`/`catch` block unless I'm missing something